packages/create-waku/package.json

```diff
@@ -25,16 +25,18 @@
     "start": "node ./dist/index.js",
     "dev": "ncc build ./src/index.ts -w -o ./dist/",
     "compile": "rm -rf template dist *.tsbuildinfo && pnpm run template && pnpm run build",
-    "template": "cp -r ../../examples template/ && rm -rf template/*/dist && rm -rf template/*/node_modules && mv template/01_template/.gitignore template/01_template/gitignore",
+    "template": "mkdir template && cp -r ../../examples/01_template template/01_template && rm -rf template/*/dist && rm -rf template/*/node_modules && mv template/01_template/.gitignore template/01_template/gitignore",
```

Comment:
Maybe we can just `cp -r ../../examples/01_template template`. You can consider it in the follow-up PRs.

Reply:
yeah i've tried to do that. but i've got it like below:

```
/template
  /src
  package.json
  ...
```

i want to do this below:
```
/template
  /01_template
```

because it will be added some templates 👍

Reply:
okay, i thought the former is fine (single template). We don't know the future, but for now we have only one template.